### DISCUSSION TITLE
Before and after. Activity Log. File modified

### DISCUSSION THREAD

**Discussion by Poseidon281** [pelican-dev/panel#275] [Ideas]
I have an idea for improving the activity log in the Pelican Panel. Currently, when a file is changed, the log only shows that a modification has occurred. I propose that the activity log should also display what exactly was changed, providing a "before and after" comparison similar to what you see on GitHub when making a commit or pull request. This would give users a clearer and more detailed view of the changes made.

**Comment by rmartinoscar** [comment]
Good idea in theory but my panel already struggles with the actual logs if i don't prune every 30 days so i really don't see it going better with such an add; I on the other hand want to implement less repetitive ActivityLog, before logging a new one check if it already exists then add new timestamp to an array and print it based on that; it would make filemanager logs way more manageable.
